docs/appendix/try-viam/rover-resources/rover-tutorial-1.md

```diff
@@ -95,7 +95,7 @@ For more information, see [Camera Component](/components/camera/).
 
 {{<imgproc src="appendix/try-viam/rover-resources/viam-rover/accelerometer.jpg" resize="400x" declaredimensions=true alt="A ADXL345 accelerometer" >}}
 
-The [ADXL345](/components/movement-sensor/adxl345/) sensor manufactured by Analog Devices is a digital 3-axis accelerometer that can read acceleration up to ±16g for high-resolution (13-bit) measurements.
+The [ADXL345](https://github.com/viam-modules/analog-devices/tree/main/adxl345) sensor manufactured by Analog Devices is a digital 3-axis accelerometer that can read acceleration up to ±16g for high-resolution (13-bit) measurements.
```

Comment:
here, and pretty much everywhere else I replaced links, I linked to the specific folder of the model in question. would you prefer I instead link to either the repo at the top level which could contain multiple models, to the readme itself, or to the section in the readme about configuring the model in questions?

Reply:
I'd prefer we link to the repo at the top level so people can see the configuring info, exactly. I can change that

Reply:
that also future proofs it if model names change etc